R-package/R/xgb.model.dt.tree.R

```diff
@@ -55,7 +62,7 @@
 #'  
 #' @export
 xgb.model.dt.tree <- function(feature_names = NULL, model = NULL, text = NULL,
-                              trees = NULL, ...){
+                              trees = NULL, use_int_id = FALSE, ...){
```

Comment:
Could you document argument changes (new argument or change of default values) to the change log?